src/components/subscriptions/LicenseAllocationNavigation.jsx

```diff
@@ -48,22 +48,25 @@ export default function LicenseAllocationNavigation() {
 
   return (
     <nav className="nav sticky-top">
-      <ul className="list-unstyled w-100">
-        {tabs.map(tab => (
-          <li key={tab.key}>
-            <button
-              className={classNames(
-                'btn btn-link btn-block pl-0 text-left',
-                { 'font-weight-bold': activeTab === tab.key },
-              )}
-              onClick={() => updateTabWithDefaultPage(tab.key)}
-              type="button"
-            >
-              {tab.text}
-            </button>
-          </li>
-        ))}
-      </ul>
+      {overview.assigned + overview.activated + overview.revoked > 0
```

Comment:
nit: maybe do an early return instead of adding this as a conditional.

```jsx

const overviewCount = overview.assigned + overview.activated + overview.revoked;
if (!overviewCount) {
  return null;
)

return (
  <nav>
    <ul className="list-unstyled w-100">
      {tabs.map(tab => (
        // ...
      ))}
  </nav>
);

```